tabular/src/autogluon/tabular/models/catboost/catboost_utils.py

```diff
@@ -66,10 +69,12 @@ def get_catboost_metric_from_ag_metric(metric, problem_type, quantile_levels=Non
         )
         metric_class = metric_map.get(metric.name, 'RMSE')
     elif problem_type == QUANTILE:
-        if quantile_levels is None or not all(0 < q < 1 for q in quantile_levels):
+        if quantile_levels is None:
+            raise AssertionError(f'quantile_levels must be provided for problem_type = {problem_type}')
+        if not all(0 < q < 1 for q in quantile_levels):
```

Comment:
better to include what the quantile_levels were in the error message, that way it is easy to visually confirm via the message that the values aren't valid

Reply:
For example:

```
f'quantile_levels must fulfill 0 < q < 1, provided quantile_values: {quantile_values}'
```